src/db/table/payment_type.sql

```diff
@@ -0,0 +1,8 @@
+-- Payment types represent the supported payment methods
+CREATE TABLE `payment_type` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `uq_name` varchar(25) NOT NULL,
```

Comment:
Oh, misunderstanding here, sorry. Maybe I shoulda worded this better. The name of the column is fine as `name`, just create a unique key called `uq_name` by adding a constraint: ```UNIQUE KEY `uq_name` (`name`)```. See #20 for examples of this. For all following occurrences as well.